features/steps/project/source/browse_files.rb

```diff
@@ -106,8 +122,35 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
   end
 
   step 'I can see new file page' do
-    expect(page).to have_content "New file"
-    expect(page).to have_content "Commit message"
+    page.should have_content "new file"
+    page.should have_content "Commit message"
+  end
+  
+  step 'I can see "upload existing one"' do
+    page.should have_content "upload existing one"
+  end
+
+  step 'I click on "upload existing one"' do
+      click_link 'upload existing one'
+  end
+  
+  step 'I click on "Upload file"' do
+    click_button 'Upload file'
+  end
+  
+  step 'I upload "user.feature"' do
+    attach_file(:file_upload, File.join('features', 'user.feature'))
+  end
+  
+  step 'I check name of the uploaded file' do
+    ".gitignore" != "user.feature"
+    "LICENSE" != "user.feature"
+    "VERSION" != "user.feature"
```

Comment:
@liyakun you are comparing two strings. The result of the comparison will _always_ be true, because these are not variables, just plain strings.

@randx is wondering why you would add this, as it doesn't help the tests. This part will always pass. It's the same as comparing `"1" != "0"`.

Did you made a mistake here or are you trying to do something else? Please explain what the purpose is.